clojure-mode.el

```diff
@@ -1888,6 +1888,11 @@ content) are considered part of the preceding sexp."
       (zero-or-one (any ":'")) ;; (in-ns 'foo) or (ns+ :user)
       (group (one-or-more (not (any "()\"" whitespace))) symbol-end)))
 
+(make-obsolete-variable 'clojure-namespace-name-regex 'clojure-namespace-regexp "5.12.0")
+
+(defconst clojure-namespace-regexp
```

Comment:
I think you've added this accidentally.

Reply:
I wasn't sure whether to change the current variable's meaning or introduce a new variable while keeping the old one around and marking it as obsolete in case some other packages currently depend on it. What do you think?

Reply:
Ah, I didn't see you actually use this. Well, I guess it's fine like this. When I scanned quickly the code I thought it had the same name as the old one. :-)